.gitignore

```
@@ -0,0 +1,4 @@
+
+node_modules/
+
+package-lock.json
```

Unneeded blank line at the top of file
```suggestion
node_modules/

package-lock.json
```